host_core/lib/host_core/claims/server.ex

```diff
@@ -10,15 +10,15 @@ defmodule HostCore.Claims.Server do
     Logger.info("Received claims command (#{cmd})")
     # PUT
     if cmd == "get" do
-      # TODO - implement claims query
-      # {:reply, claims}
+      # Subscription failure
     else
       claims = Msgpax.unpack!(body) |> Map.new(fn {k, v} -> {String.to_atom(k), v} end)
-      key = claims.public_key
+      # Note the name difference here, outside of OTP the claims use JWT naming conventions
+      key = claims.sub
       :ets.insert(:claims_table, {key, claims})
 
-      if claims.call_alias != nil && String.length(claims.call_alias) > 0 do
-        :ets.insert(:callalias_table, {claims.call_alias, claims.public_key})
+      if claims.call_alias != nil && String.length(claims.call_alias) > 1 do
+        :ets.insert(:callalias_table, {claims.call_alias, claims.sub})
```

Comment:
Are we making a choice here that call aliases need to be more than 1 character? (Don't get me wrong, I don't mind. if I saw someone call an actor named `X` i'd cringe a bit)

Reply:
yes, I want to make sure it's greater than 1 character. I ran a test that accidentally set the call alias to a newline and everything caught fire, including me.

Reply:
I had a weird suspicion that this was a newline issue. Cool 👍🏻